commit ef3d5bcbd02abb4994fbc8dc2f248e9213ae2a1e (refs/changes/67/2867/2)
Author: Todd Whiteman <todd.whiteman@joyent.com>
Date:   2017-10-25T13:18:20-07:00 (1 year, 11 months ago)
    
    DOCKER-1120 test cli-attach fails when it has to download the ubuntu image
    Reviewed by: Dave Eddy <dave.eddy@joyent.com>
    Approved by: Dave Eddy <dave.eddy@joyent.com>

diff --git a/test/integration/cli-attach.test.js b/test/integration/cli-attach.test.js
index 6624a86..29be986 100644
--- a/test/integration/cli-attach.test.js
+++ b/test/integration/cli-attach.test.js
@@ -57,6 +57,16 @@ test('setup', function (tt) {
             t.end();
         });
     });
+
+    tt.test('pull ubuntu image', function (t) {
+        h.ensureImage({
+            name: 'ubuntu:latest',
+            user: ALICE
+        }, function (err) {
+            t.error(err, 'should be no error pulling image');
+            t.end();
+        });
+    });
 });
 
 
